pulp_rpm/app/modulemd.py

```diff
@@ -87,6 +87,7 @@ def parse_modulemd(module_names, module_index):
             modulemd[PULP_MODULE_ATTR.NAME] = stream.props.module_name
             modulemd[PULP_MODULE_ATTR.STREAM] = stream.props.stream_name
             modulemd[PULP_MODULE_ATTR.VERSION] = stream.props.version
+            modulemd[PULP_MODULE_ATTR.STATIC_CONTEXT] = stream.props.static_context
```

Comment:
I do not know what to do here.
This line will fail if the new enough libmodulemd is unavailable, at the same time, I do not know how to require libmodulemd of a certain version in python requirements.

Checking the presence of the `static_context` attribute feels like a bad idea, because we'll pretend to work and potentially set the wrong static context.

Reply:
I don't think there's much we can do about this other than to assume that a compatible libmodulemd is present.  We should verify that RHEL / CentOS 7 and 8 have the appropriate updates.

Reply:
Used getattr(,,None) to handle "what if the attribute isn't known to libmodulemd?"